test/e2e/external_gateways.go

```diff
@@ -3005,7 +3005,7 @@ func reachPodFromGateway(targetAddress, targetPort, targetPodName, srcContainer,
 
 	res, err := runCommand(dockerCmd...)
 	framework.ExpectNoError(err, "Failed to reach pod %s (%s) from external container %s", targetAddress, protocol, srcContainer)
-	framework.ExpectEqual(strings.Trim(res, "\n"), targetPodName)
+	gomega.Expect(strings.Trim(res, "\n")).To(gomega.Equal(targetPodName))
```

Comment:
is framework.ExceptEqual just gone? commit didn't pint to relevant changes in k/k where that happened.